src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final String SUPPLY_WORD = "supply";
+    static final String BUY_WORD = "buy";
+    static final String RESULT_WORD = "result";
+    static final int NAME_COLUMN_INDEX = 0;
+    static final int VALUE_COLUMN_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(getFromFile(fromFileName),toFileName);
```

Comment:
I would separate this method invocation like this:

```
String fileData = readFromFile(fromFileName);
String result = processData(fileData);
writeToFile(result, toFileName);
```

Also, you have method writeToFile, so appropriate naming from read from file will be readFromFile.
Better to use one style for naming and formatting.
It's minor remark.